### DISCUSSION TITLE
Holopins for contributors

### DISCUSSION THREAD

**Discussion by bran0h** [kodadot/nft-gallery#4362] [General]
Maybe you should consider to be part of [holopin](https://www.holopin.io/) and give some specials rewards to some contributors

**Comment by elenalape** [comment]
👀

**Reply by yangwao** [reply]
Mmm

**Reply by spacesmutje** [reply]
Don't do Mmmm. You already made your decision 😃

**Reply by yangwao** [reply]
<img src="https://media0.giphy.com/media/KYdR9Uno6et3LQcEDU/giphy.gif"/>

**Comment by yangwao** [comment]
Oh well, found some bugs @elenalape  ?  

https://github.com/kodadot/nft-gallery/issues/4625

![image](https://user-images.githubusercontent.com/5887929/211156171-7f803b8f-ad7f-4ecb-a6a7-faf4fb68d1fa.png)

Installed, authenticated, added holopin.yml, can't create organization apparently? 

https://github.com/kodadot/nft-gallery/blob/main/.github/holopin.yml

only instruction I've found
![image](https://user-images.githubusercontent.com/5887929/211156238-529c0383-00e7-4403-b47b-a6ca56f21f7a.png)
